CarpetX/src/driver.cxx

```diff
@@ -2575,6 +2581,32 @@ CCTK_INT CarpetX_GetBoundarySizesAndTypes(
   return 0;
 }
 
+const int *ArrayGroupSizeB(const cGH *cctkGH, int dir, int gi,
+                           const char *groupname) {
+  if (groupname) {
+    gi = CCTK_GroupIndex(groupname);
+  }
+  assert(gi >= 0 and gi < CCTK_NumGroups());
```

Comment:
We're now using `&&` instead of `and`...

Reply:
Done